app/src/views/private/components/revisions-drawer-detail/revision-item.vue

```diff
@@ -163,6 +163,11 @@ export default defineComponent({
 	}
 
 	.user {
+		span {
+			margin: -6px;
+			padding: 6px;
+		}
```

Comment:
Good call to increase the hitbox for this a notch too 👍🏻

Reply:
Thanks :) when I look/think back at it, the thought of making the hitbox larger suddenly popped up 😄 used to do the same thing for buttons-look-like-links in past projects